README.md

```diff
@@ -39,11 +44,27 @@ Further documentation is available on [our docs site](https://docs.ironcorelabs.
 
 Prerequisites:
 
-+ [Rust toolchain installed](https://www.rust-lang.org/tools/install)
-+ `JAVA_HOME` environment variable set
-+ `clang` installed
+- [Rust toolchain](https://www.rust-lang.org/tools/install) installed
+- `JAVA_HOME` environment variable set
+- `clang` installed
+
+From the root of this repository run `cargo build -p ironoxide-java`. The resulting `target/debug/build/ironoxide-java-*/out/java` directory will have the JNI binding code for the Java side and `target/debug` will have the dynamic library file you need to pull into your Java code. It will be named `libironoxide_java.so` or `libironoxide_java.dylib` depending on your environment. This library will only work on the architecture from which it was built.
+
+# IronOxide-Android
+
+## Build from Source
+
+Prerequisites:
```

Comment:
These aren't correct, but I can fix that up in my PR

Reply:
I guess that I did remove the `JAVA_HOME` stuff, so that makes sense. Are there other parts not required?

And yeah, I figured you'd know more about this section and be able to fill it in as part of your PR.